spec/aptible/cli/subcommands/db_spec.rb

```diff
@@ -439,6 +439,75 @@ def expect_provision_database(create_opts, provision_opts = {})
     end
   end
 
+  describe '#db:dump' do
+    it 'should fail if database is non-existent' do
+      allow(Aptible::Api::Database).to receive(:all) { [] }
+      expect do
+        subject.send('db:dump', handle)
+      end.to raise_error("Could not find database #{handle}")
+    end
+
+    context 'valid database' do
+      before do
+        allow(Aptible::Api::Database).to receive(:all) { [database] }
+        allow_any_instance_of(Aptible::CLI::Agent).to receive(:`)
```

Comment:
Use `allow(subject)` instead of `allow_any_instance_of(Aptible::CLI::Agent)`

Reply:
Done